source/fundamentals/linq.txt

```diff
@@ -320,6 +320,77 @@ following documents:
    { "date" : ISODate("2012-12-05T00:00:00Z"), "grade" : "A", "score" : 13 }
    { "date" : ISODate("2012-05-17T00:00:00Z"), "grade" : "A", "score" : 11 }
 
+Nested Statements
++++++++++++++++++
+
+You can chain or nest ``Select`` and ``SelectMany`` statements to unwind nested
+arrays. Consider a collection that holds documents with a ``restaurants`` field.
+This field contains an array of documents that represent the ``Restaurants`` class:
+
+.. code-block:: json
+
+   { 
+      "_id": { "$oid": ... },
+      "restaurants": [
+         {
+            "_id": { ... } ,
+            "address": { ... },
+            "name": "Tov Kosher Kitchen",
+            "grades": [
+               { 
+                  "date" : ISODate("2014-11-24T00:00:00Z"), 
+                  "grade" : "Z", 
+                  "score" : 20.0 
+               }, 
+               { 
+                  "date" : ISODate("2013-01-17T00:00:00Z"), 
+                  "grade" : "A", 
+                  "score" : 13.0 
+               },
+               ...
+            ]
+            ...
+         },
+         {
+            "_id": { ... } ,
+            "address": { ... },
+            "name": "Harriet's Kitchen",
+            "grades": [ ... ],
+            ...
+         },
+         ...
+      ]
+   }
+
+You can nest ``SelectMany`` statements within ``SelectMany`` or ``Select``
+statements. The following example nests a ``SelectMany`` statement within a
+``Select`` statement to retrieve an array of ``Grade`` documents from all restaurants
+in the document above.
```

Comment:
S: "preceding" instead of "[above](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#above)":
```suggestion
in the preceding document.
```